docs/src/vertical_grids.md

```diff
@@ -50,20 +50,40 @@ axislegend(ax, position=:rb)
 fig
 ```
 
+Let's see how [`ExponentialFaces`](@ref) works:
```

Comment:
```suggestion
Let's see how [`ExponentialInterfaces`](@ref) works:
```

can we use "interfaces" instead of "faces"?

Reply:
agreed; done